runtime/lua/vim/_meta/builtin.lua

```diff
@@ -194,10 +194,10 @@ function vim.str_utfindex(str, index) end
 --- @return string|nil Converted string if conversion succeeds, `nil` otherwise.
 function vim.iconv(str, from, to, opts) end
 
---- Schedules {callback} to be invoked soon by the main event-loop. Useful
+--- Schedules {fn} to be invoked soon by the main event-loop. Useful
 --- to avoid |textlock| or other temporary restrictions.
---- @param callback fun()
-function vim.schedule(callback) end
+--- @param fn fun()
```

Comment:
```suggestion
--- @param fn function
```

Reply:
Seems to be a few of those around:

```
runtime/lua/vim/treesitter/languagetree.lua|187 col 4| ---@param f fun(): R1, R2, R2
runtime/lua/vim/_system.lua|235 col 5| --- @param on_error fun()
runtime/lua/vim/_system.lua|247 col 4| ---@param cb fun()
runtime/lua/vim/_meta/builtin.lua|199 col 5| --- @param fn fun()
runtime/lua/vim/_meta/builtin.lua|235 col 5| --- @param callback? fun(): boolean Optional callback. Waits until {callback} returns true
runtime/lua/vim/_meta/builtin.lua|283 col 5| --- @param callback fun()
runtime/lua/vim/_meta/misc.lua|13 col 5| --- @param func fun()
```

Want me to patch them all?

Reply:
No. A lot/all of those are intentional and are used by LuaLs.